linux_os/guide/system/software/integrity/software-integrity/aide/file_audit_tools_permissions/rule.yml

```diff
@@ -0,0 +1,59 @@
+documentation_complete: true
+
+prodtype: rhel8,rhel9
+
+title: Audit Tools Must Have a Mode of 0755 or Less Permissive
+
+description: |-
+    {{{ full_name }}} systems providing tools to interface with audit information will leverage user permissions and roles identifying the user accessing the tools, and the corresponding rights the user enjoys, to make access decisions regarding the access to audit tools.
+
+    Audit tools include, but are not limited to, vendor-provided and open source audit tools needed to successfully view and manipulate audit information system activity and records. Audit tools include custom queries and report generators.
+
+    Audit tools must have the file_permissions.
+
+rationale: |-
+    Protecting audit information also includes identifying and protecting the tools used to view and manipulate log data.
+    Therefore, protecting audit tools is necessary to prevent unauthorized operations on audit information.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: CCE-86227-6
+    cce@rhel9: CCE-86228-4
+
+references:
+    disa: CCI-001493
+    nist: AU-9
+    srg: SRG-OS-000256-GPOS-00097
+    stigid@rhel8: RHEL-08-030620
+
+ocil_clause: 'any of these files have more permissive permissions than 0755'
+
+ocil: |-
+    Verify the audit tools are protected from unauthorized access, deletion, or modification by checking the permissive mode.
+
+    Check the octal permission of each audit tool by running the following command:
+
+    $ sudo stat -c "%U %n" /sbin/auditctl /sbin/aureport /sbin/ausearch /sbin/autrace /sbin/auditd /sbin/rsyslogd /sbin/augenrules
+
+fix: |-
+    Configure the audit tools to be owned by "root", by running the following command:
```

Comment:
to have mode 0755 instead of owned by root

the whole fixtext is for the other rule